lassen/asm.py

```diff
@@ -152,3 +152,6 @@ def sgt():
 def sge():
     return inst(ALU.Sub, cond=Cond.SGE)
 
+def const(val):
+    return inst(ALU.Add,ra_mode=Mode.CONST,ra_const=val)
```

Comment:
Seems like not the right assembly name for this...? Or the right abstraction in general?

Perhaps a better way to do this would be something like a qualifier:

```python
# Takes in instruction @inst and sets reg @reg to be const, with val @val 
def const(inst, reg, val):
    ...
```

One difficulty might be the fact that instructions are unhashable but perhaps there is some way to get around this? @cdonovick

Reply:
@rsetaluri, not sure what you are asking about. This is an abstraction that returns an instruction which makes the PE into a constant with a certain value. This is not a way to fold a constant into an already existing instruction. I will have a separate way to fold a constant. This is to unblock PnR by providing a naive way to map constants